documentation/3.3/content/userguide/aks/_index.md

```diff
@@ -296,6 +297,31 @@ If you want to use this template to automate the deployment, download it by sele
 
 Click **Create** to create this offer. This process may take 30 to 60 minutes.
 
+#### Template outputs
+
+After clicking **Create** to create this offer, you will go to the **Deployment is in progress** page. When the deploymet is completed, the page shows **Your deployment is complete**. In the left panel, select **Outputs**. These are the outputs from the deployment.  The following table is a reference guide to the deployment outputs.
+
+| Field | Description |
+|-------|-------------|
+| `aksClusterName` | Name of your AKS cluster that is running the WLS cluster. {{< line_break >}}Sample value: `wlsonaksiyiql2i2o2u2i`. |
+| `adminConsoleInternalUrl` | The fully qualified, private link to the Administration Console portal. You can only access it inside AKS cluster. {{< line_break >}}Sample value: `http://sample-domain1-admin-server.sample-domain1-ns.svc.cluster.local:7001/console`. |
```

Comment:
can only access it -> can access it only 
inside AKS cluster -> inside an AKS cluster